docker/scripts/dev_start.sh

```diff
@@ -239,14 +239,12 @@ function determine_dev_image() {
     local version="$1"
     # If no custom version specified
     if [ -z "${version}" ]; then
-        if [ "${USE_LOCAL_IMAGE}" = "yes" ]; then
-            version="local_dev"
-        elif [ "${TARGET_ARCH}" = "x86_64" ]; then
+        if [ "${TARGET_ARCH}" = "x86_64" ]; then
             version="${VERSION_X86_64}"
         elif [ "${TARGET_ARCH}" = "aarch64" ]; then
             version="${VERSION_AARCH64}"
         else
```

Comment:
In this PR: when `dev_start.sh -l` was run,  It no longer searches for the `local_dev` image. Instead, image tag for that arch is used.  If someone do need the `local_dev` tag, he/she should just run `dev_start.sh -l -t local_dev`.